docs/HYPER.md

```diff
@@ -34,9 +34,17 @@ Build curl to use hyper's C API:
      % git clone https://github.com/curl/curl
      % cd curl
      % autoreconf -fi
-     % ./configure --with-hyper=<hyper dir>
+     % ./configure --with-hyper=<hyper-dir>
      % make
 
+And run the tests:
+
+     % make tests
+
+If you get a link error saying `libhyper.so` was not found, prefix the
+`./configure` command with `LDFLAGS="-Wl,-rpath,<hyper-dir>/target/debug"`
+and/or `LDFLAGS="-Wl,-rpath,<hyper-dir>/target/release"`.
```

Comment:
Maybe a simpler approach is to instead just unconditionally add this to the configure line furher up?

Reply:
Good suggestion. I have updated accordingly.